custom_components/eventsensor/config_flow.py

```diff
@@ -21,6 +21,8 @@
     PRESET_HUE_DIMMER_MAPPING,
     PRESET_HUE_TAP,
     PRESET_HUE_TAP_MAPPING,
+    PRESET_HUE_BUTTON,
+    PRESET_HUE_BUTTON_MAPPING,
```

Comment:
@clarkd, when you have the dev setup for HA (for the other PR), then run `` in here ;-), that will say something like ` flake8 failed, I101 Imported names are in the wrong order` :)

It's my fault because I have not enabled CI in this repo (the `.pre-commit-config.yaml` file is present, but nothing ensures it is installed)

Anyway, it was just a minor detail to comment, the PR is perfectly ok, so it can be merged anytime. I'll try to combine with the other one opened right now to publish a new release for HACS

You could also fill the CHANGELOG and bump the minor version of the repo :)

Reply:
I'm using the devcontainer for HA, and because this is a different repo I don't think I can just run that pre-commit command? Any tips?

Reply:
In the meantime, I've fixed the ordering and updated the changelog as well as the version in pyproject.toml.